app/parse.py

```diff
@@ -15,8 +27,57 @@ class Product:
     num_of_reviews: int
 
 
+def get_products_from_page(driver: WebDriver, url: str) -> list[Product]:
+    driver.get(url)
+    time.sleep(1)
```

Comment:
Explicit vs Implicit Waits: Using time.sleep() to wait for page elements can cause inefficiency and errors, especially if the page takes more or less time to load. Instead, Selenium's WebDriverWait is recommended, which waits for specific elements or conditions.